PhysicsTools/PatAlgos/python/producersLayer1/patCandidates_cff.py

```diff
@@ -37,9 +37,21 @@
 
 _patCandidatesTask = patCandidatesTask.copy()
 _patCandidatesTask.remove(makePatOOTPhotonsTask)
-from Configuration.ProcessModifiers.pp_on_AA_cff import pp_on_AA
-pp_on_AA.toReplaceWith(patCandidatesTask, _patCandidatesTask) 
-pp_on_AA.toModify(patCandidateSummary.candidates, func = lambda list: list.remove(cms.InputTag("patOOTPhotons")) )
+from Configuration.Eras.Modifier_pp_on_AA_2018_cff import pp_on_AA_2018
+from Configuration.Eras.Modifier_pp_on_PbPb_run3_cff import pp_on_PbPb_run3
+HI = (pp_on_AA_2018 | pp_on_PbPb_run3)
+HI.toReplaceWith(patCandidatesTask,
+                 patCandidatesTask.copyAndExclude([makePatOOTPhotonsTask]))
+HI.toModify(patCandidateSummary.candidates,
+            func = lambda list: list.remove(cms.InputTag("patOOTPhotons")) )
+
+from Configuration.Eras.Modifier_run2_miniAOD_94XFall17_cff import run2_miniAOD_94XFall17
+from Configuration.Eras.Modifier_run2_miniAOD_80XLegacy_cff import run2_miniAOD_80XLegacy
+mAOD = (run2_miniAOD_94XFall17 | run2_miniAOD_80XLegacy)
```

Comment:
```suggestion
_mAOD = (run2_miniAOD_94XFall17 | run2_miniAOD_80XLegacy)
```
to avoid exporting this symbol

Reply:
Done.